quarkus/pom.xml

```diff
@@ -18,15 +18,12 @@
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
         <maven.compiler.release>17</maven.compiler.release>
-        <quarkus.version>2.13.7.Final</quarkus.version>
+        <quarkus.version>3.0.0.CR2</quarkus.version>
```

Comment:
nitpicking: property can be move to build/configuration/pom.xml

Reply:
Moved and I've updated the property to use the `version.%s` format.